keras/preprocessing/image.py

```diff
@@ -580,8 +582,8 @@ def random_transform(self, x):
 
         if shear != 0:
             shear_matrix = np.array([[1, -np.sin(shear), 0],
-                                    [0, np.cos(shear), 0],
-                                    [0, 0, 1]])
+                                     [0, np.cos(shear), 0],
+                                     [0, 0, 1]])
```

Comment:
The indent is meant to match to upper logical line, not the semantics of np arrays